### PR TITLE
Add dictonary definition for OSS

### PR DESCRIPTION
**Pull request by laurasimsdev**
## This PR fixes...

The lack of dictionary definition and perspective for Open Source Software

## What I did...

- Add dictionary definition for OSS
- Add Software Engineer perspective for OSS

## How to test...

_In a list format, tell us how to test this PR._
Ex:

1. Navigate to /dictionary
2. Search "open source software"
3. See if definition and perspective show up under "open source software"
4. Check for grammar and typos

## I learned...

About the preview function in GitHub flavored markdown

### CODE REVIEW

File: src/assets/content/dictionary/open-source-software-oss.md [hunk 1]
```diff
@@ -1,6 +1,12 @@
 ---
 title: Open Source Software (OSS)
-definition: ''
-perspectives: []
+definition: Open source software is software that is available for anyone to view, copy, change, or distribute for any purpose. Some notable examples of open source software include the Linux operating system, the Mozilla Firefox browser, Apache HTTP Server software, and LibreOffice productivity software. Every year, open source software contributions are celebrated during October as part of a month-long event called Hacktoberfest
```
Comment (alliequintano) [inline src/assets/content/dictionary/open-source-software-oss.md]:
```suggestion
definition: Open source software is software that is available for anyone to view, copy, change, or distribute for any purpose. Some notable examples of open source software include the Linux operating system, the Mozilla Firefox browser, Apache HTTP Server software, and LibreOffice productivity software. Every year, open source software contributions are celebrated during October as part of a month-long event called Hacktoberfest.
```

File: src/assets/content/dictionary/open-source-software-oss.md [hunk 2]
```diff
@@ -1,6 +1,12 @@
...
+
+sources: 
+- sourceurl: https://en.wikipedia.org/wiki/Open-source_software, https://hacktoberfest.com
+perspectives: 
+- meaning: is an important part of the software development community, allowing collaboration of large numbers of programmers with diverse perspectives, increasing the speed of innovation
```
Comment (alliequintano) [inline src/assets/content/dictionary/open-source-software-oss.md]: Hi @laurasimsdev ! Thank you so much for your contribution to our tech dictionary. I have just a couple small request to get your changes ready for approval.

Please take a look at our [CONTRIBUTING.md](https://github.com/cherryontech/website/blob/pit/CONTRIBUTING.md) to see the necessary formatting. The `meaning` field will be a part of an existing sentence structure, so we can see the perspective sentence doesn't read quite right when we test your change (via Netlify):
image

I strongly suggest following the directions listed in the CONTRIBUTING.md for testing locally or via the Netlify deploy preview in order to read your sentence. This helps you catch mistakes like the additional period and capitalization, as well as reading the full sentence for clarity.

**Review by alliequintano — CHANGES_REQUESTED** [review]
Please see my comments and add me as a reviewer when you're ready for re-review! Thank you.

**Comment by laurasimsdev** [comment]
I made the requested updates. Thanks!

**Review by alliequintano — APPROVED** [review]
Great job with the changes 👍 Thanks again for contributing to our tech dictionary!

**Pull request merged**